taskcluster/fenix_taskgraph/transforms/build.py

```diff
@@ -9,6 +9,7 @@
 from __future__ import absolute_import, print_function, unicode_literals
 
 import datetime
+import os
```

Comment:
`add_release_version()` calls `os.environ`. It shows `add_release_version()` wasn't called before.